vaadin-testbench-core-junit5/src/main/java/com/vaadin/testbench/AbstractBrowserTestBase.java

```diff
@@ -32,36 +29,17 @@
  * A superclass with helper methods to aid TestBench developers create a JUnit 5
  * based tests.
  */
-public abstract class TestBenchTestBase
+public abstract class AbstractBrowserTestBase
         implements HasDriver, HasTestBenchCommandExecutor, HasElementQuery {
 
-    @RegisterExtension
-    public ScreenshotOnFailureExtension screenshotOnFailureExtension = new ScreenshotOnFailureExtension(
-            this, true);
-
-    protected WebDriver driver;
-
-    protected Capabilities capabilities;
-
-    @BeforeEach
-    public void initializeWebDriveAndCapabilities(WebDriver driver,
-            Capabilities capabilities) {
-        this.driver = driver;
-        this.capabilities = capabilities;
-    }
-
     /**
-     * Returns the {@link WebDriver} instance previously specified within
-     * {@link #initializeWebDriveAndCapabilities(WebDriver, Capabilities)}, or
-     * (if the previously provided WebDriver instance was not already a
-     * {@link TestBenchDriverProxy} instance) a {@link TestBenchDriverProxy}
-     * that wraps that driver.
+     * Returns the {@link WebDriver} instance or (if the previously provided
```

Comment:
This description gives an implementation details of the AbstractBrowserDriverTestBase, but could the implementation of this method be different?